src/commands.ts

```diff
@@ -13,28 +13,22 @@ export function getCommand(agent: Agent, commnad: Command, args: string[]) {
   return c.replace('{0}', args.join(' '))
 }
 
-export function parseNi(agent: Agent, _args: string[]): string {
-  let command: Command = 'install'
-  let args: string[] = []
+export function parseNi(agent: Agent, args: string[]): string {
+  if (args.length === 0)
+    return getCommand(agent, 'install', [])
 
-  if (_args.length === 0) {
-    command = 'install'
-    args = []
-  }
-  else if (_args.includes('-g')) {
-    command = 'global'
-    args = exclude(_args, '-g')
-  }
-  else if (_args.includes('--frozen')) {
-    command = 'frozen'
-    args = exclude(_args, '--frozen')
-  }
-  else {
-    command = 'add'
-    args = _args
+  if (args.includes('-g'))
+    return getCommand(agent, 'global', exclude(args, '-g'))
+
+  if (args.includes('--frozen'))
+    return getCommand(agent, 'frozen', exclude(args, '--frozen'))
+
+  if (args.includes('--frozen=safe')) {
```

Comment:
Thinking if the name is clear enough? maybe `--frozen-if-present`?

Reply:
I agree it's confusing. I was mixing it with "fail-safe".

`frozen-if-present` sounds good to me!